infra/terraform/test-org/org/locals.tf

```diff
@@ -752,5 +752,23 @@ locals {
       owners      = ["anaik91", "imrannayer"]
       topics      = local.common_topics.db
     },
+    {
+      name        = "terraform-pubsub-integration-golang"
+      short_name  = "pubsub-golang-app"
+      org         = "GoogleCloudPlatform"
+      description = "Reliable low-latency server-to-server communication around the globe"
+      owners      = ["anguillanneuf", "balajismaniam", "donmccasland", "msampathkumar"]
+      # groups          = ["dee-data-ai", local.jss_common_group]
```

Comment:
Please take these out if not needed. Although we've been enabling periodic testing for all JSS solutions so you may want to keep that.

Same comments for the repo below.